devel/ansible/roles/hotness-dev/tasks/main.yml

```diff
@@ -45,14 +48,24 @@
 
 - name: Install the-new-hotness package into /usr/lib
   pip:
-    name: /home/{{ ansible_env.SUDO_USER }}/devel/
+    name: "{{ item }}"
     extra_args: '-e'
+    executable: pip-3
+  with_items:
+    - "file:///home/{{ ansible_env.SUDO_USER }}/devel/"
+    - "file:///home/{{ ansible_env.SUDO_USER }}/devel/hotness_schema/"
 
-- name: Install fedmsg config to ~/.fedmsg.d
+- name: Install fedora-messaging config to ~/
   become_user: "{{ ansible_env.SUDO_USER }}"
   copy:
-    src: fedmsg.d/
-    dest: /home/{{ ansible_env.SUDO_USER }}/.fedmsg.d
+    src: config.toml
+    dest: /home/{{ ansible_env.SUDO_USER }}/
+
+- name: Enable RabbitMQ server
+  command: systemctl enable rabbitmq-server
+
+- name: Start RabbitMQ server
+  command: systemctl start rabbitmq-server
```

Comment:
You could also use [the systemd module](https://docs.ansible.com/ansible/latest/modules/systemd_module.html#systemd-module) for those two.

Reply:
Good point